src/spaghetti-and-meatballs.md

```diff
@@ -82,6 +84,6 @@
 
 ## Contribution
 
-- ClosedGL - [GitHub](https://github.com/ClosedGL2)
+- ClosedGL -- [GitHub](https://github.com/ClosedGL2), [PeerTube](https://truetube.media/https://truetube.media/video-channels/cgl/videos)
```

Comment:
```suggestion
- ClosedGL -- [GitHub](https://github.com/ClosedGL2), [PeerTube](https://truetube.media/video-channels/cgl/videos)
```

Looks like you have `truetube.media` twice in the url

Reply:
Fixed